content/en/docs/developer-documentation/build-install-container.md

```diff
@@ -38,6 +38,16 @@ Build the MicroShift AIO container:
 ```Bash
 make microshift-aio
 ```
+## Tagging the Image
+
+It is necessary to tag the image as latest to ensure that the newly created image is used when starting MicroShift with systemd.
+
+```Bash
+IMAGE=$(podman images | grep micro | awk '{print $3}')
+podman tag ${IMAGE} quay.io/microshift/microshift:latest
+```
+
+
 
 ## Running the MicroShift Containers
```

Comment:
This section also needs an update - but that can be a follow-up, I want to say something like 
To run containerized, refer to [podman section from getting-started](blah#blah).
If runnint the AIO, refer to [the aio section](blah#blah).

Reply:
Included in https://github.com/redhat-et/microshift-documentation/pull/105